### PR TITLE
[6122] Pending approval and schedule for publish showing info of the last modifier of the item instead of the current one

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6122

### CODE REVIEW

**Comment by jvega190** [comment]
Waiting for addition of `dateSubmitted` prop

File: ui/app/src/utils/content.ts [hunk 1]
```diff
@@ -272,11 +279,18 @@ export function parseLegacyItemToDetailedItem(item: LegacyItem | LegacyItem[]):
     sandbox: {
       creator: null,
       dateCreated: null,
-      modifier: item.user,
+      modifier: {
+        username: item.user,
+        firstName: item.user,
+        lastName: item.user,
```
Comment (rart) [inline ui/app/src/utils/content.ts]:
```suggestion
        firstName: null,
        lastName: null,
```

File: ui/app/src/utils/content.ts [hunk 2]
```diff
@@ -251,11 +251,18 @@ export function parseLegacyItemToSandBoxItem(item: LegacyItem | LegacyItem[]): S
     ...parseLegacyItemToBaseItem(item),
     creator: null,
     dateCreated: null,
-    modifier: item.user,
+    modifier: {
+      username: item.user,
+      firstName: item.user,
+      lastName: item.user,
```
Comment (rart) [inline ui/app/src/utils/content.ts]:
```suggestion
      firstName: null,
      lastName: null,
```

File: ui/app/src/state/epics/content.ts [hunk 3]
```diff
@@ -326,7 +326,9 @@ const content: CrafterCMSEpic[] = [
     action$.pipe(
       ofType(conditionallyUnlockItem.type),
       withLatestFrom(state$),
-      filter(([{ payload }, state]) => state.content.itemsByPath[payload.path].lockOwner === state.user.username),
+      filter(
+        ([{ payload }, state]) => state.content.itemsByPath[payload.path].lockOwner.username === state.user.username
```
Comment (rart) [inline ui/app/src/state/epics/content.ts]:
```suggestion
        ([{ payload }, state]) => state.content.itemsByPath[payload.path].lockOwner?.username === state.user.username
```

**Review by rart — CHANGES_REQUESTED** [review]
Add a `CHANGELOG.md` to `ui/app/src` and let's start tracking changes.

**Comment by jvega190** [comment]
@rart re-requested a review but I haven't added the changelog, I'll work on it

**Review by rart — APPROVED** [review]

**Comment by jvega190** [comment]
This PR needs to be merged with https://github.com/craftercms/studio/pull/3276 to avoid issues on the UI

**Pull request merged**